docs/reference-manual/native-image/assets/build-output-schema-v1.json

```diff
@@ -0,0 +1,586 @@
+                    "default": {},
+                    "title": "The garbage collection resource usage schema",
+                    "required": [
+                        "count",
+                        "total_secs"
+                    ],
+                    "properties": {
+                        "count": {
+                            "type": "integer",
+                            "default": 0,
+                            "title": "The number of GC cycles performed during image generation",
+                            "examples": [
+                                17
+                            ]
+                        },
+                        "total_secs": {
+                            "type": "number",
+                            "default": 0.0,
+                            "title": "The total time spent in GC (in seconds)",
+                            "examples": [
+                                0.9245
+                            ]
+                        }
+                    },
+                    "examples": [{
+                        "count": 17,
+                        "total_secs": 0.9245
+                    }]
+                },
+                "memory": {
+                    "type": "object",
+                    "default": {},
+                    "title": "The memory statistics schema",
+                    "required": [
+                        "system_total",
+                        "peak_rss_bytes"
+                    ],
+                    "properties": {
+                        "system_total": {
+                            "type": "integer",
+                            "default": 0,
+                            "title": "The total system memory",
+                            "examples": [
+                                33254146048
+                            ]
+                        },
+                        "peak_rss_bytes": {
+                            "type": "integer",
+                            "default": 0,
+                            "title": "Peak RSS value of the image generator process in bytes",
+                            "examples": [
+                                3506065408
+                            ]
+                        }
+                    },
+                    "examples": [{
+                        "system_total": 33254146048,
+                        "peak_rss_bytes": 3506065408
+                    }]
+                }
+            },
+            "examples": [{
+                "cpu": {
+                    "load": 8.38,
+                    "total_cores": 10
+                },
+                "garbage_collection": {
+                    "count": 17,
+                    "total_secs": 0.9245
+                },
+                "memory": {
+                    "system_total": 33254146048,
+                    "peak_rss_bytes": 3506065408
+                }
+            }]
+        }
+    },
+    "examples": [{
+        "general_info": {
```

Comment:
Do we really need to include this many sub-examples if there is one full example? When are sub-examples useful or could we maintain one full example instead?